rocAL/source/loaders/image/image_loader_sharded.cpp

```diff
@@ -162,6 +162,13 @@ Timing ImageLoaderSharded::timing() {
     return t;
 }
 
+size_t ImageLoaderSharded::last_batch_padded_size() {
+    size_t sum = 0;
+    for (auto& loader : _loaders)
```

Comment:
Why does this have to be sum of all loaders?

Reply:
@SundarRajan28 / @sbavasab any specific reason?

Reply:
As discussed, changed the implementation